### PR TITLE
🚮 Remove `rsimha` from all owners files and comments

### PR DESCRIPTION
**Pull request by rsimha**
👋⚡🏗️😢

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -9,7 +9,6 @@
         {name: 'cramforce', requestReviews: false},
         {name: 'dvoytenko', requestReviews: false},
         {name: 'jridgewell', requestReviews: false},
-        {name: 'rsimha', requestReviews: false},
       ],
```
Comment (danielrozenberg) [inline OWNERS:12]: Should probably add `newmuis` and/or me here?

File: third_party/OWNERS [hunk 2]
```diff
@@ -4,12 +4,7 @@
 {
   rules: [
     {
-      owners: [
-        {name: 'rsimha'},
-        {name: 'erwinmombay'},
-        {name: 'choumx'},
-        {name: 'jridgewell'},
-      ],
+      owners: [{name: 'erwinmombay'}, {name: 'choumx'}, {name: 'jridgewell'}],
```
Comment (danielrozenberg) [inline third_party/OWNERS:7]: Maybe remove `choumx` while you're here?

File: OWNERS (hunk 1 above)
Comment (rsimha) [reply OWNERS:12]: SGTM, but let's do this in a separate PR for tracking.

File: third_party/OWNERS (hunk 2 above)
Comment (rsimha) [reply third_party/OWNERS:7]: Codesearch shows dozens of instances where owners files contain folks who have left the team. I'll leave this for a separate cleanup.

File: OWNERS (hunk 1 above)
Comment (danielrozenberg) [reply OWNERS:12]: This should be done in the same PR, because we'd have one less global approver to make this change otherwise

Comment (rsimha) [reply OWNERS:12]: Adding a new repo-wide owner should be a separate action in its own PR created by the new owner and approved by existing owners. Piggybacking on a clean up PR makes it difficult to track. Admins can do this without the need for an extra global approver.

**Review by danielrozenberg — APPROVED** [review]

**Pull request merged**